docs/ref/reference.rst

```diff
@@ -119,6 +120,11 @@ commands are dealing with the monitor:
      You may use the `--nodename` command line option to bypass the whole
      DNS lookup based process and force the local node name to a fixed
      value.
+     
+     The ``--auth`` option allows setting up authentication method to be used
+     for connections from data nodes with ``autoctl_node`` user.
+     If this option to be used, please make sure password is manually set on
+     the monitor, and appropriate setting is added to `.pgpass` file on data node.      
```

Comment:
I think we should expand on the `.pgpass` idea with a list of HBA rules needed for pg_auto_failover:

  - the monitor health check, from the monitor to the Postgres nodes,
  - the pg_autoctl connections from the Postgres nodes to the monitor, using the `autoctl_node` user,
  - the replication connection from the secondary to the primary,
  - the replication connection after a failover, so from the current primary to the current secondary,
  - what about replacing a node with a new one?

Reply:
added a paragraph on .pgpass on Security section